### ISSUE TITLE
Problem with the command line "ln -s ../../src/dist/ ."

### ISSUE THREAD

**Issue by mariela21180** [jaumard/ngx-dashboard#31]
Me dice: ""ln" no se reconoce como un comando interno o externo,
programa o archivo por lotes ejecutable."
No se como solucionarlo.

**Comment by jaumard** [comment]
Hello,

This command work only in Mac and Linux system, if you are on windows you need to find the right command to create a symlink

**Comment by mariela21180** [comment]
Oh, I see...

After many, many tests, I get to do it. The Windows command is:

mklink /D dist ..\..\src

I had to use the cmd.exe run as administrator, and use the "src" folder, because with "src\dist" there was an error that diddn't find the modules.

You can add this to the documentation if you want, in case it helps someone else

**Comment by jaumard** [comment]
Glad you make it work :) your command is strange but I don't know nothing about windows ^^ I added the command on the readme. Thanks for sharing !

**Comment by mariela21180** [comment]
Opps, there is a typo, sorry.
It's 
mklink /D dist ..\..\src

**Comment by mariela21180** [comment]
Opps, there is a typo, sorry. Seems like Github re-write it from .. \ .. \ to ....\ Here is an image of the correct command:

![image](https://user-images.githubusercontent.com/3533830/30178536-ab00bb0a-93df-11e7-9263-ec3fb5158ee4.png)

Yes, the command is strange, and I had to try many ways until one worked haha.
On windows seems like we have to say it in the opposite way (create this in here, instead of create here this), and with the bar \ not /, and instead of create the "dist" symlink on the "../../src/dir" I had to create it on "../../src" for it to detect the modules.

Glad to share! :)
Thanks to you for publish the dashboard!
